content/ru/docs/reference/kubectl/kubectl.md

```diff
@@ -0,0 +1,566 @@
+---
+title: kubectl
+content_template: templates/tool-reference
+weight: 28
+---
```

Comment:
This is [generated documentation](https://kubernetes.io/docs/contribute/generate-ref-docs/kubernetes-components/)

If you'd like to, you can contribute localized text for kubectl [upstream](https://github.com/kubernetes/kubernetes/); see https://github.com/kubernetes/kubernetes/blob/master/translations/README.md

Reply:
Oh, I didn’t know about it, so the translation of this document cannot be accepted? :(

Reply:
You can translate, but it has to be upstream. Otherwise the next time someone generates documentation your hard work is lost.

Reply:
I get it, then we'll accept this translation now, but I'll do translation in the upstream repository later, okay?

Reply:
> we'll accept this translation now, but I'll do translation in the upstream repository later

Reply:
It's a pity! So I need to revert the file or all files at all?

Reply:
Just this file. And please do save your work, it is still useful to have the translated text available. I hope you will contribute upstream.

Reply:
So it might be better to commit this file so that it is stored in this repository (useful for contributing to upstream and just so as not to lose current work)? It may also be useful for other contributors, for example.